src/components/common/Sidebar/Sidebar.js

```diff
@@ -69,21 +69,20 @@ const Sidebar = props => {
   };
 
   // This is determining which role is currently in session, implemented further in ternary statements in the return clause
-  const role = localStorage.role_id;
   const isUserMentee = () => {
-    if (role === '4') {
+    if (role_id === '4') {
```

Comment:
I like that this has been changed from role to role_id, this consistency makes sense to me